.circleci/config.yml

```diff
@@ -518,26 +518,34 @@ jobs:
                 circleci-agent step halt
             fi
       - run:
-          name: Create the distribution files
-          command: |
-            make schemas_dist
-      - run:
-          name: Upload to PyPI
-          command: |
-            # Relies on the TWINE_USERNAME and TWINE_PASSWORD environment variables configured at:
-            #   https://app.circleci.com/settings/project/github/mozilla/experimenter/environment-variables
-            # For more on twine, see:
-            #   https://twine.readthedocs.io/en/latest/
-            make schemas_deploy_pypi SCHEMAS_ENV="-e TWINE_USERNAME=${TWINE_USERNAME} -e TWINE_PASSWORD=${TWINE_PASSWORD}"
-      - run:
-          name: NPM Authentication
-          command: echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npmrc
-      - run:
-          name: Upload to NPM
-          command: |
-            # Relies on the NPM_TOKEN environment variable configured at:
-            #   https://app.circleci.com/settings/project/github/mozilla/experimenter/environment-variables
-            make schemas_deploy_npm SCHEMAS_ENV="-e NPM_TOKEN=${NPM_TOKEN}"
+          name: Build the distribution files
+          command: |
+            make schemas_build
```

Comment:
Do we still need this as a separate step since it's a dep to the dist now?

Reply:
No probably not :)